### PR TITLE
Updated for alpine image and other changes

### PR DESCRIPTION
**Pull request by geraldoandradee**
In this PR:

* Change base image to alpine (smaller and consequently faster to deploy);
* Fix files permissions during creation;
* Fix documentation about how to create a new project (the old one create file as root permissions);
* Ignoring some files with updated `.gitignore` and `.dockerignore`;
* Changed app port from 80 to 8080 (@Nilsty See if I miss something, please)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *.pyc
 test_service
 .pytest_cache
+.idea
```
Comment (Menda) [inline .gitignore]: Include this in your local global gitignore. Not everyone is using the same editor: https://help.github.com/articles/ignoring-files/

File: README.md [hunk 2]
```diff
@@ -12,10 +12,11 @@ project will be created for Python version 3
 
 ## Set up a new MicroService
 
-Just run the command:
+Just run the commands:
 
 ```bash
-docker-compose run --rm django_service_wizard
+$ docker build . -t django_service_wizard
+$ docker run -it -u $(id -u):$(id -g) -v "$(pwd)":/code django_service_wizard
```
Comment (Menda) [inline README.md]: This looks way more complicated than before. I don't see the reason to modify how it was before

File: service_builder/setup_service.py [hunk 3]
```diff
@@ -155,23 +155,24 @@ def _create_app(name_project: str, name_app: str):
...
-    for (src, dest) in src_dest_list:
+    for (src, dest, perm) in src_dest_list:
         if dest != '.' and not os.path.isdir(dest):
             os.mkdir(dest)
         content = get_template_content(src)
         content = content.replace('{{ name_project }}', name_project)
         append_to_file(
             os.path.join(dest, os.path.basename(src)),
-            content, recreate=True)
+            content, recreate=True, permission=perm)
```
Comment (Menda) [inline service_builder/setup_service.py]: Why complicate it more adding permissions?

File: service_builder/setup_service.py [hunk 4]
```diff
@@ -184,37 +185,23 @@ def _configure_docker(name_project: str):
...
 
-    for file in files_to_copy:
+    for source, destination, permission in files_to_copy:
         append_to_file(
-            filename=file['destination'],
-            text_to_append=get_template_content(file['source']),
+            filename=destination,
+            text_to_append=get_template_content(source),
             recreate=True,
+            permission=permission
```
Comment (Menda) [inline service_builder/setup_service.py]: Same thing.

File: service_builder/setup_service.py [hunk 5]
```diff
@@ -229,7 +216,8 @@ def _configure_docker_registry(name_project: str, registry_domain: str,
     content = get_template_content(filename)
     content = content.replace('{{ registry_domain }}', registry_domain)
     content = content.replace('{{ registry_url }}', registry_url)
-    append_to_file(filename='.drone.yml', text_to_append=content)
+    append_to_file(filename='.drone.yml', text_to_append=content,
+                   permission=0o644)
```
Comment (Menda) [inline service_builder/setup_service.py]: And same thing.

File: service_builder/tests/test_setup_service.py [hunk 6]
```diff
@@ -190,7 +190,7 @@ def test_configure_docker(self):
              'python manage.py collectstatic --no-input'),
             (os.path.join('scripts', 'run-tests.sh'),
              'python manage.py makemigrations --check --dry-run'),
-            (os.path.join('scripts', 'wait-for-it.sh'), 'wait-for-it')
+            # (os.path.join('scripts', 'wait-for-it.sh'), 'wait-for-it')
```
Comment (Menda) [inline service_builder/tests/test_setup_service.py]: Why is this commented?

File: service_builder/tests/expected/scripts/wait-for-it.sh [hunk 7]
```diff
@@ -1,179 +1,23 @@
...
-    -s | --strict               Only execute subcommand if the test succeeds
-    -q | --quiet                Don't output any status messages
-    -t TIMEOUT | --timeout=TIMEOUT
-                                Timeout in seconds, zero for no timeout
-    -- COMMAND ARGS             Execute command with args after the test finishes
-USAGE
+if [[ -z "$1" || -z "$2" ]]
+then
+    echo "tcp-port-wait - block until specified TCP port becomes available"
+    echo "Usage: tcp-port-wait HOST PORT"
```
Comment (Menda) [inline service_builder/tests/expected/scripts/wait-for-it.sh]: The name of the script doesn't correspond to this.

File: service_builder/templates/drone-ci/.drone.yml [hunk 8]
```diff
@@ -8,9 +8,10 @@ services:
 
 pipeline:
   test:
-    image: python:3.6
+    image: python:3.6-alpine3.7
     commands:
-      - bash scripts/wait-for-it.sh $${DATABASE_HOST}:$${DATABASE_PORT}
+      - pip install -r requirements/ci.txt --no-cache-dir
```
Comment (Menda) [inline service_builder/templates/drone-ci/.drone.yml]: `bash scripts/run-tests.sh --ci` already installs `requirements/ci.txt`

File: service_builder/templates/gitignore/.gitignore [hunk 9]
```diff
@@ -1 +1,106 @@
...
+
+# Rope project settings
+.ropeproject
+
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
+.idea
```
Comment (Menda) [inline service_builder/templates/gitignore/.gitignore]: This is way too much! It's up to the developer to add its own files. The minimal before IMHO was fine.

**Comment by Nilsty** [comment]
the change from port 80 to 8080 looks good to me.

File: README.md (hunk 2 above)
Comment (geraldoandradee) [reply README.md]: The reason is the ownership of the created files in host machine. All files are created as `root`. With the new command line the file permissions are created correctly as a non root user.

File: .gitignore (hunk 1 above)
Comment (geraldoandradee) [reply .gitignore]: Removed.

File: service_builder/setup_service.py (hunk 3 above)
Comment (geraldoandradee) [reply service_builder/setup_service.py]: It's not complicating per se. The file permissions were wrong when created without permissions. Well at least in linux was created incorrectly. I'm sure in Mac OS was wrong too.

File: service_builder/tests/test_setup_service.py (hunk 6 above)
Comment (geraldoandradee) [reply service_builder/tests/test_setup_service.py]: Typo. Removed comment. File is created in other place.

File: service_builder/tests/expected/scripts/wait-for-it.sh (hunk 7 above)
Comment (geraldoandradee) [reply service_builder/tests/expected/scripts/wait-for-it.sh]: Fixed

File: service_builder/setup_service.py (hunk 4 above)
Comment (geraldoandradee) [reply service_builder/setup_service.py]: Same thing.

File: service_builder/setup_service.py (hunk 5 above)
Comment (geraldoandradee) [reply service_builder/setup_service.py]: And same thing.

File: service_builder/templates/gitignore/.gitignore (hunk 9 above)
Comment (geraldoandradee) [reply service_builder/templates/gitignore/.gitignore]: I've made it smaller. But not so smaller. :)

**Pull request merged**